go.mod

```diff
@@ -88,9 +88,9 @@ require (
 	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575
 	github.com/cilium/ebpf v0.11.0
 	github.com/clbanning/mxj v1.8.4
-	github.com/containerd/cgroups v1.1.0
```

Comment:
nit: Ideally, if we can move the `indirect` to the `indirect` group below, that's cleaner

Reply:
good point, let me fix this